abi/abi.go

```diff
@@ -295,14 +295,9 @@ func FromValues(vals []*Value) []interface{} {
 func Deserialize(data []byte, t Type) (*Value, error) {
 	switch t {
 	case Address:
-		addr, err := address.NewFromBytes(data)
-		if err != nil {
-			return nil, err
-		}
-
 		return &Value{
 			Type: t,
-			Val:  addr,
+			Val:  address.NewFromBytes(data),
```

Comment:
NewFromBytes should probably still return an error. This is parsing user input, panicking is not a good thing